tests/Unit/Product/WCProductAdapterTest.php

```diff
@@ -255,6 +255,42 @@ public function test_description_is_set() {
 		$this->assertEquals( $expected_description, $adapted_product->getDescription() );
 	}
 
+	public function test_description_shortcodes_are_applied() {
+		add_filter(
+			'woocommerce_gla_product_description_apply_shortcodes',
+			function () {
+				return true;
+			}
```

Comment:
I guess it'a a matter of preference, but WordPress has the `__return_true` function for this.

Reply:
Yeah, I try to depend on WP as little as possible when it comes to tests to make them more isolated.